utils.h

```diff
@@ -109,7 +110,7 @@ static inline int strcmp0(const char *str1, const char *str2)
 	if (!str2)
 		return 1;
 
-	return strcmp(str1, str2);
+	return strnatcmp(str1, str2);
```

Comment:
OK, I see — https://github.com/jayay/cmus/commit/2cc7836f069a0a2e85db688e4b9c77bb78fea552.

However, please create a separate `strnatcmp0` and use that instead. `strcmp0` is also used in `lib.c`.